spring-kafka-docs/src/main/asciidoc/retrytopic.adoc

```diff
@@ -0,0 +1,449 @@
+[[retry-topic]]
+== Non-Blocking Retries
+
```

Comment:
I think we should add (for now).

IMPORTANT: This is an experimental feature and the usual rule of no breaking API changes does not apply to this feature until the experimental designation is removed.
Users are encouraged to try out the feature and provide feedback via GitHib Issues or GitHub discussions.

Reply:
Sure, I totally agree with that, done.